FHEM/14_Hideki.pm

```diff
@@ -136,73 +136,72 @@ Hideki_Parse($$)
 	}elsif($sensorTyp==13){
 		($channel, $temp) = decodeThermo(\@decodedData); # decodeThermoHygro($decodedString);
 		$bat = ($decodedData[2] >> 6 == 3) ? 'ok' : 'low';			 # decode battery
 		$count = $decodedData[3] >> 6;		# UNVERIFIZIERT, MSG_Counter
 		$val = "T: $temp";
-		Log3 $iohash, 4, "$name decoded Hideki protocol model=$model, sensor id=$id, channel=$channel, cnt=$count, bat=$bat, temp=$temp";
-		Log3 $iohash, 4, "$name Sensor Typ $sensorTyp currently not full supported, please report sensor information!";
+		Log3 $iohash, 4, "$ioname decoded Hideki protocol model=$model, sensor id=$id, channel=$channel, cnt=$count, bat=$bat, temp=$temp";
+		Log3 $iohash, 4, "$ioname Sensor Typ $sensorTyp currently not full supported, please report sensor information!";
 	}
 	else{
-		Log3 $iohash, 4, "$name Sensor Typ $sensorTyp not supported, please report sensor information!";
+		Log3 $iohash, 4, "$ioname Sensor Typ $sensorTyp not supported, please report sensor information!";
 		return "";
 	}
 	my $longids = AttrVal($iohash->{NAME},'longids',0);
 	if ( ($longids ne "0") && ($longids eq "1" || $longids eq "ALL" || (",$longids," =~ m/,$model,/)))
 	{
 		$deviceCode=$model . "_" . $id . "." . $channel;
-		Log3 $iohash,4, "$name Hideki_Parse: using longid: $longids model: $model";
+		Log3 $iohash,4, "$ioname Hideki_Parse: using longid: $longids model: $model";
 	} else {
 		$deviceCode = $model . "_" . $channel;
 	}
 
-	Log3 $iohash, 5, "$name Hideki_Parse deviceCode: $deviceCode";
+	Log3 $iohash, 5, "$ioname Hideki_Parse deviceCode: $deviceCode";
 
 	my $def = $modules{Hideki}{defptr}{$iohash->{NAME} . "." . $deviceCode};
 	$def = $modules{Hideki}{defptr}{$deviceCode} if(!$def);
 
 	if(!$def) {
-		Log3 $iohash, 1, "$name Hideki: UNDEFINED sensor $deviceCode detected, code $msg";
+		Log3 $iohash, 1, "$ioname Hideki: UNDEFINED sensor $deviceCode detected, code $msg";
 		return "UNDEFINED $deviceCode Hideki $deviceCode";
 	}
 
 	my $hash = $def;
-	$name = $hash->{NAME};
+	my $name = $hash->{NAME};
 	return "" if(IsIgnored($name));
 
 	#Log3 $name, 4, "Hideki: $name ($msg)";
 	
-	my $WindSpeedCorr = AttrVal($name,"windSpeedCorr",0);
-	if ($WindSpeedCorr > 0 && $sensorTyp == 12) {
-		$windspeed = sprintf("%.2f", $windspeed * $WindSpeedCorr);
-		$windgust  = sprintf("%.2f", $windgust * $WindSpeedCorr);
-		Log3 $name, 4, "$name Hideki_Parse: WindSpeedCorr=$WindSpeedCorr, WindSpeed=$windspeed, WindGust=$windgust";
+	if ($sensorTyp == 12) {	# Wind
+		($windchill,$windspeed,$windgust,$winddir,$winddirdeg,$winddirtext) = wind($name, \@decodedData);
+		$val = "T: $temp  Ws: $windspeed  Wg: $windgust  Wd: $winddirtext";
+		Log3 $name, 4, "$ioname $name Parse: model=12(wind), T: $temp, Wc=$windchill, Ws=$windspeed, Wg=$windgust, Wd=$winddir, WdDeg=$winddirdeg, Wdtxt=$winddirtext";
 	}
 	
-	if (!defined(AttrVal($hash->{NAME},"event-min-interval",undef)))
+	if (!defined(AttrVal($ioname,"event-min-interval",undef)))
```

Comment:
Das ist nicht richtig, wir haben an dieser Stelle die Definition bereits identifiziert und wollen dann auch von dieser die passenden Attribute verwenden:

```suggestion
	if (!defined(AttrVal($name,"event-min-interval",undef)))
```

Reply:
Hier hast du Recht. Ich sehe , das ein paar Zeilen oben drüber der name definiert wird.
Hast du die Änderung in den Branch schon eingebracht? Ich sehe noch keine Anpassung welche du oben als Änderung zeigtest.

Im Branch ist immer noch 
`if (!defined(AttrVal($ioname,"event-min-interval",undef)))`

Reply:
Ich habe es als Änderungsvorschlag hier hinterlegt, der in der Github WebUI angenommen oder abgelehnt werden kann:
![image](https://user-images.githubusercontent.com/7968127/173435375-35225185-b87a-4daf-9afb-da1e22a31590.png)